log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -238,5 +245,6 @@ public void configure(LoggingConfiguration config)
                 throw new UncheckedIOException(e);
             }
         }
+        configured = true;
```

Comment:
Move this to right after the if block at the beginning of the method. If something fails with an exception, we still need to flag that the configure() method has been called.